src/content/docs/user/quickstart.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Quickstart
+---
+
+1. Install the `ricochet` [CLI](https://github.com/ricochet-cli/cli)
+
+1. Login into a ricochet instance ([try.ricochet.rs](https://try.ricochet.rs) is our public demo instance):
+
+   ```bash
+   export RICOCHET_SERVER=https://try.ricochet.rs
+   ricochet login
+   ```
+
+1. Deploy a new App using the `deploy` command:
+
+   ```bash
+   # clone example app
+   git clone https://github.com/rstudio/shiny-examples.git /tmp/shiny-examples
+
+   # navigate to the app directory
+   cd /tmp/shiny-examples/050-kmeans-example/
+   ```
+
+   ```bash
+   ricochet deploy
+   ```
```

Comment:
work noting that this wont work if there isn't a `_ricochet.toml` file. 

Some ideas: 
- We can curate our own repo of test / example apps which have a `_ricochet.toml` file. 
- We can add the `ricochet init` call to `ricochet deploy` if no `_ricochet.toml` file is found
- Add a step to `ricochet init` first